build.gradle

```diff
@@ -15,16 +15,25 @@ configurations {
 	runtimeClasspath {
 		extendsFrom developmentOnly
 	}
+	implementation {
+		exclude group: 'org.slf4j', module: 'slf4j-log4j12'
```

Comment:
Out of curiosity, what is the reason for excluding org.slf4j here?

Reply:
Because sl4j complains that `Class path contains multiple SLF4J bindings`. I chose the one used by spring.